main.c

```diff
@@ -18,8 +18,8 @@
  */
 
 #include "nsxiv.h"
-#include "commands.h"
 #define _MAPPINGS_CONFIG
+#include "commands.h"
 #include "config.h"
 
```

Comment:
It has been my opinion for config.h to include the needed headers itself instead of just assuming they are present. If we sorted this include list, we wouldn't compile which seems less than ideal. At the very least we should probably comment on config.h's assumptions on what is already included.

Reply:
> include the needed headers itself instead of just assuming they are present

I think we do that for all the other headers. Special casing config.h seems ok to me, not a fan of including headers in what's supposed to be a configuration file for the users.